combat-trainer.lic

```diff
@@ -1829,7 +1829,11 @@ class AbilityProcess
     @barb_buffs = @buffs.delete('barb_buffs') || []
     echo("  @barb_buffs: #{@barb_buffs}") if $debug_mode_ct
 
-    setup_barb_buff_flags
+    @barb_buffs_inner_fire_threshold = settings.barb_buffs_inner_fire_threshold
+    echo("  @barb_buffs_inner_fire_threshold: #{@barb_buffs_inner_fire_threshold}") if $debug_mode_ct
+
+    @meditation_pause_timer = settings.meditation_pause_timer
+    echo("  @meditation_pause_timer: #{@meditation_pause_timer}") if $debug_mode_ct
```

Comment:
Existing setting in base.yaml, but new to being used in combat-trainer. Replaces a hard coded 6 seconds to wait for a meditation to take effect.